src/System.Data.SqlClient/tests/FunctionalTests/ExceptionTest.cs

```diff
@@ -138,25 +138,6 @@ public void NamedPipeInvalidConnStringTest_ManagedSNI()
             builder.DataSource = @"np:\\\\\";
             OpenBadConnection(builder.ConnectionString, invalidConnStringError);
         }
-
-        [Fact]
-        public static void LocalDBNotSupportedOnUapTest()
```

Comment:
Do you have any idea how this test was determining that the platform was Uap? I don't see any attributes decorating the method.